api/internal/routes.go

```diff
@@ -90,7 +104,7 @@ func (router *Router) Routes() {
 					WithDescription("Enter your JWT token in the format: Bearer <token>"),
 			},
 		}),
-		fuego.WithAddr(":"+PORT),
+		fuego.WithAddr(serverAddr),
```

Comment:
this change might break the api container when run in production with docker compose file, test this out with priority

Reply:
noted, I'll run the installation script locally and verify, that would do right?